scripts/superdesk-authoring/authoring.js

```diff
@@ -61,9 +61,9 @@
 
     var DEFAULT_SCHEMA = Object.freeze({
         slugline: {maxlength: 24},
-        headline: {maxlength: 64, order: 1, editor: ['h1', 'h2', 'bold', 'italic', 'underline', 'quote', 'anchor']},
-        abstract: {maxlength: 160, order: 2, editor: ['h1', 'h2', 'bold', 'italic', 'underline', 'quote', 'anchor']},
-        body_html: {order: 5, editor: ['h1', 'h2', 'bold', 'italic', 'underline', 'quote', 'anchor', 'embed', 'picture', 'table']},
```

Comment:
haha, we understand each other better and better, but I still don't know if you agree that this a problem that values of buttons can change.
My first thought is that it shouldn't change. If you define them in `content_type.json` then it should show them at the first rendering. Because now I have the feeling it uses the default values, and if the directive is reloaded again, it will use the values from `content_type.json` (for PA it means empty)
Maybe this occurs only in the case when you didn't specified the buttons list in `content_type.json`...
Is it the wanted behaviour when `content_type.json` doesn't provide the button list ?